From 975a1f2feb486922f586e69d119b27d05d5ab38a Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Mon, 17 Feb 2020 11:39:06 +0100
Subject: [PATCH] Fix AppData file

XML declaration is allowed only at the start of the document.
---
 data/me.mitya57.ReText.appdata.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/me.mitya57.ReText.appdata.xml b/data/me.mitya57.ReText.appdata.xml
index 8ba01a2..9851f43 100644
--- a/data/me.mitya57.ReText.appdata.xml
+++ b/data/me.mitya57.ReText.appdata.xml
@@ -1,5 +1,5 @@
-<!-- Copyright 2018 Dmitry Shachnev -->
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Dmitry Shachnev -->
 <component type="desktop">
   <id>me.mitya57.ReText</id>
   <name>ReText</name>
@@ -57,7 +57,7 @@
   <translation type="qt">retext</translation>
   <developer_name>Dmitry Shachnev</developer_name>
   <update_contact>mitya57_AT_gmail.com</update_contact>
-  <content_rating type="oars-1.0" />
+  <content_rating type="oars-1.0"/>
   <releases>
     <release version="7.0.4" date="2018-09-23"/>
     <release version="7.0.3" date="2018-06-06"/>
